Art/mintera10/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cute Puppy Animation</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="puppy">
+        <div class="puppy-body">
+            <div class="puppy-ear left"></div>
+            <div class="puppy-ear right"></div>
+            <div class="puppy-eye left"></div>
+            <div class="puppy-eye right"></div>
+            <div class="puppy-nose"></div>
+            <div class="puppy-tail"></div>
+        </div>
+        <div class="puppy-leg front-left"></div>
+        <div class="puppy-leg front-right"></div>
+        <div class="puppy-leg back-left"></div>
+        <div class="puppy-leg back-right"></div>
+    </div>
+</body>
+</html>
+<body>
+    <img src="images/puppy-animation.gif">
```

Comment:
## Invalid source & Invalid implementation maybe
1. HTML issue: You cannot have 2 bodies in HTML
2. Wrong Contribution approach: it seems like you want to display your own gif within your own implementation:
the gif is to be provided in the folder ( looks like you did it with puppy.gif in mintera10 folder ) 
whereas the source image here is referring to an inner folder images that does not exist.

Please remove the whole second body.